readme.txt

```diff
@@ -49,6 +49,9 @@ This is compatible with WooCommerce>=2.4, including the new 3.0 release. It has
 
 = 1.0.2 =
```

Comment:
I think now, we are supporting latest release of razorpay-woocommerce, so it should be minor 'or' major release.
@harman28 please suggest.